metricbeat/module/system/process_summary/process_summary_test.go

```diff
@@ -0,0 +1,37 @@
+// +build darwin freebsd linux windows
+
+package process_summary
+
+import (
+	"testing"
+
+	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestData(t *testing.T) {
+	f := mbtest.NewEventFetcher(t, getConfig())
+	err := mbtest.WriteEvent(f, t)
+	if err != nil {
+		t.Fatal("write", err)
+	}
+}
+
+func TestFetch(t *testing.T) {
+	f := mbtest.NewEventFetcher(t, getConfig())
+	event, err := f.Fetch()
+	assert.NoError(t, err)
+	assert.Contains(t, event, "sleeping")
+	assert.Contains(t, event, "running")
+	assert.Contains(t, event, "idle")
+	assert.Contains(t, event, "stopped")
+	assert.Contains(t, event, "zombie")
+	assert.Contains(t, event, "unknown")
```

Comment:
Missing `total`. 

And how about a test to ensure the sum of the various states equals `total`?